xtp/include/votca/xtp/orbitals.h

```diff
@@ -390,11 +424,17 @@ class Orbitals {
   bool useTDA_;
 
   tools::EigenSystem mos_;
+  tools::EigenSystem mos_embedding_;
+
+  Eigen::MatrixXd pm_localized_orbitals_;
+  Index active_electrons_;
```

Comment:
I am not sure, if this is ideal, because you couple orbitals to some specific qmmm method, but time will tell.